src/main/java/by/custom/utilcalculator/service/Bot.java

```diff
@@ -196,23 +201,23 @@ private String getStringOptionsForEngineVolume() {
     private String getStringOptionsForAgeAuto() {
         return "." +
                 "\n" +
-                Storage.AGE_OF_AUTO;
+                Questions.AGE_OF_AUTO;
     }
 
     private String getStringOptionsForPrice() {
```

Comment:
I proceed to see `String` in many methods. Is it really needed?

Reply:
We don't need it, sorry, my fault - didn't clean it yestarday. Today's fix contains less ...String... methods names. I still have two methods with this words, but I will try to fix it too.